### ISSUE TITLE
Additional issue with mc 4.8.30

### ISSUE THREAD

**Issue by snhirsch** [yurytch/mc.ext_to_mc.ext.ini#5]
FYI: For 4.8.30 it's also necessary to double backslashes.  After running your awk script:

`sudo sed -i '/Regex=/s/\\/\\\\/g'  /etc/mc/mc.ext.ini`

Apparently a change in one of the dependent libraries caused the problem.  It's supposed to be fixed in 4.8.31, FWIW.

**Comment by yurytch** [comment]
Have you an example of things not working as expected with backslashes not doubled? I've never noticed anything of the kind.

**Comment by snhirsch** [comment]
Yes, I do.  If you convert (or write from scratch) an mc version 4 ext file and try to use it with mc 4.8.30 as packaged in Ubuntu 24.04 you will find that none of the regex based rules work.  After doubling up on the slashes it will then operate.  Apparently this is a result of changes to glib.  You can find discussion here:

https://forums.linuxmint.com/viewtopic.php?t=426497

**Comment by yurytch** [comment]
Thank you, I see. However, it's not caused by my thingy, so what do you suggest I do? 
I think I'll just add the issue's description to `README`.
